s3.tf

```diff
@@ -1,5 +1,5 @@
-resource "aws_s3_bucket" "devops_s3_course_2022_moved_resource" {
-  bucket = "rady-bucket-unique-name"
+resource "aws_s3_bucket" "dzmitry_s3" {
```

Comment:
make the resource name more generic so i can merge it to the repo and others can use it. somethinmg like "devops_2022_s3".
Also clean up the # from the bucket name below